From: Michael Biebl <biebl@debian.org>
Date: Thu, 18 Jul 2013 15:33:51 +0200
Subject: Add note to udev.conf that changes to that file require a rebuild of
 the initramfs

Based on debian/patches/udev_conf_comments from the old udev package.

(cherry picked from commit f6de1b02fe703f57cceb59922fde17385e7c4510)
---
 src/udev/udev.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/udev/udev.conf b/src/udev/udev.conf
index 47d1433..0d812d4 100644
--- a/src/udev/udev.conf
+++ b/src/udev/udev.conf
@@ -1,3 +1,6 @@
 # see udev.conf(5) for details
+#
+# udevd is also started in the initrd.  When this file is modified you might
+# also want to rebuild the initrd, so that it will include the modified configuration.
 
 #udev_log="info"
